compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.cpp

```diff
@@ -278,10 +278,24 @@ void setTranslationInfo(IREE::HAL::ExecutableExportOp exportOp,
 // operations.
 // ===----------------------------------------------------------------------===//
 
+Operation *getLoweringConfigCarryingOp(ArrayRef<Operation *> computeOps) {
+  for (Operation *op : computeOps) {
+    if (getLoweringConfig(op)) return op;
+  }
+  return nullptr;
+}
+
 IREE::Codegen::LoweringConfigAttr getLoweringConfig(Operation *op) {
   return op->getAttrOfType<IREE::Codegen::LoweringConfigAttr>(kConfigAttrName);
 }
 
+IREE::Codegen::LoweringConfigAttr getLoweringConfig(
```

Comment:
If possible make this an `inline ` function in the header file.

Reply:
I'm hitting linker errors due to duplicated symbols if doing that.

Reply:
Strange.